File: revision/15s2/17.pl
#!/usr/local/bin/perl -w



$string1 = $ARGV[0];
$string2 = $ARGV[1];

if ($string1 eq $string2) {
    print "$string1\n";
} else {    
    if ($string1 =~ /(.*?)(\d+)(.*)/) {
        ($bef, $lower, $aft) = ($1, $2, $3);
    }
    if ($string2 =~ /(\d+)/) {
        $higher = $1;
    }
    $i = $lower;
    while ($i <= $higher) {
        $line = "$bef$i$aft";
        print "$line\n";
        $i++;
    }
}

File: revision/15s2/17.py
#!/usr/bin/python3

import sys, re

string1 = sys.argv[1]
string2 = sys.argv[2]

if (string1 == string2):
    print(string1)
else:
    match = re.search(r'(.*?)(\d+)(.*)', string1)
    if match:
        bef = match.group(1)
        lower = int(match.group(2))
        aft = match.group(3)
    match = re.search(r'(\d+)', string2)
    if match:
        higher = int(match.group(1))
    i = lower
    while (i <= higher):
        line = bef+str(i)+aft
        print (line)
        i += 1
  
        
